docs/auth/usage/index.md

```diff
@@ -147,7 +147,7 @@ then signing them in.
 import auth from '@react-native-firebase/auth';
 
 auth()
-  .createUserWithEmailAndPassword('sarah.lane@gmail.com', 'SuperSecretPassword!')
+  .createUserWithEmailAndPassword('Jane.Doe@example.com', 'SuperSecretPassword!')
```

Comment:
Generally, I've always gone with the lowercased email address with other examples - I rarely see or type emails with capitals.

Reply:
yeah, that makes sense